### PR TITLE
Done and delete

### PR DESCRIPTION
**Pull request by mariaalouisaa**
I really struggled to incorporate the delete and cross out functionality at first. Actually had to make some changes to the function you wrote Peter to make it work... hope that's okay! (I couldn't get my head around that input inside the input, it wasn't allowing me to make any changes to it... I only coded it would so if we have time we can still do that later)

What I've done so far:
- Add a delete button (with a trash can icon)
- Add a checkbox for done items (I have added a button but coded out if we decide to go with that instead?)
- Given each its own eventlistener and function
- Added the delete functionality for each individual items
- Toggle a strike through class when checkbox is clicked (I tried to give detailed comments on how this works, its basically a border top with ALOT of CSS used to manipulate it - only way I could find to animate it)
- Added aria-labels for accessibility

Lastly, please merge the other two branches first to avoid code being overwritten (: 

I hope this message makes sense, I'm super sleepy haha!

### CODE REVIEW

**Review by AsmahanM — COMMENTED** [review]
Thanks so much for your hard work, Maria!

File: index.js [hunk 1]
```diff
@@ -83,21 +83,61 @@ inputBtn.addEventListener('click', (e) => {
...
-  newFeatureFilm.classList.add('new-film');
-  newFeatureFilm.value = inputTitle.value;
-  newFeatureFilm.setAttribute('readonly', 'readonly');
+  // const newFeatureFilm = document.createElement('input');
+  // newFeatureFilm.classList.add('new-film');
+  // newFeatureFilm.value = inputTitle.value;
+  // newFeatureFilm.setAttribute('readonly', 'readonly');
+  
+  // Add movie title to the list
+  const movieTitle = document.createElement('p');
```
Comment (PJSalter) [inline index.js:92]: Must more readable, thank you for the change ✨

File: index.js [hunk 2]
```diff
@@ -83,21 +83,61 @@ inputBtn.addEventListener('click', (e) => {
...
+  bingeWorthy.appendChild(movieTitle);
+  
   //console.log(newFeatureFilm);
+  
+  //create done button
+  //const doneButton = document.createElement('button');
+  //doneButton.innerHTML = `<i class="fas fa-check"></i>`;
+  //doneButton.style.color = '#0ed600'
+  //doneButton.style.borderColor = '#0ed600'
+  const doneButton = document.createElement('input');
```
Comment (PJSalter) [inline index.js:104]: Awesome now we have a checkbox 😸

File: index.js [hunk 3]
```diff
@@ -83,21 +83,61 @@ inputBtn.addEventListener('click', (e) => {
...
+  doneButton.type = 'checkbox'
+  doneButton.classList.add('checkbox')
+  doneButton.ariaLabel = `Check movie title as watched`
+  doneButton.addEventListener('click', markDone)
+  bingeWorthy.appendChild(doneButton);
+
+  //create delete button
+  const deleteButton = document.createElement('button');
```
Comment (PJSalter) [inline index.js:112]: Nicely done 🤩

File: index.js [hunk 4]
```diff
@@ -83,21 +83,61 @@ inputBtn.addEventListener('click', (e) => {
...
+  inputDiv.appendChild(bingeWorthy)
 
 // append the bingeWorthy and newFeatureFilm to the inputDiv
-inputDiv.append(bingeWorthy, newFeatureFilm)
+// inputDiv.append(bingeWorthy, newFeatureFilm, deleteButton)
 
 // clearing the input title field
 //console.log(inputDiv.value = "");
-inputTitle.value = "";
+inputTitle.value = ``;
```
Comment (PJSalter) [inline index.js:128]: why the change in ticks?

File: style.css [hunk 5]
```diff
@@ -288,3 +278,41 @@ h1 {
...
+/* Strike through movie title */
+
+.movie-title {
+  color: white;
+  width: 90%;
+  position: relative;
+ overflow: hidden;  /* hides the strikethrough on default */
+}
+
+.movie-title::after { /* creates a pseudo-element that is a child of the movie title */
```
Comment (PJSalter) [inline style.css:304]: ✨ great work

**Review by PJSalter — APPROVED** [review]
